api/filter_dimension_options.go

```diff
@@ -541,7 +549,8 @@ func (api *FilterAPI) patchFilterBlueprintDimension(ctx context.Context, filterB
 
 	// apply patch operations sequentially, stop processing if one patch fails, and return a list of successful patches operations
 	for _, patch := range patches {
-		options := removeDuplicateAndEmptyOptions(patch.Value)
+		allOptions := getStringArrayFromInterface(patch.Value)
```

Comment:
if `getStringArrayFromInterface` returns an error, we should check it here and abort (it should not happen because this should have already failed before, during validation):
```
allOptions, err := getStringArrayFromInterface(patch.Value)
if err != nil {
    return successful, eTag, err
}
```

Reply:
Thanks for review.
Improvement applied.